web_src/js/features/repo-issue.js

```diff
@@ -480,7 +480,7 @@ export function initRepoPullRequestReview() {
     $(this).closest('.menu').toggle('visible');
   });
 
-  $('a.add-code-comment').on('click', async function (e) {
+  $(document).on('click', 'a.add-code-comment', async function (e) {
```

Comment:
Maybe we should add a comment here detailing why it is better to use a document listener here than a local listener, i.e.
```javascript
// We listen on the document instead of locally as files can be lazy-loaded at a later time
// and the listener would need to be added there as well
```
Otherwise, this issue might reoccur somewhen.